packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -45,13 +45,37 @@ export default class Generate extends Command {
       await setUIFramework(appName, answers.uiFramework);
 
       // Clean files after all steps
-      this.cleanFiles(appName);
+      await this.cleanFiles(appName);
+
+      // Display a final message
+      this.displayEndMessage(appName);
     } catch (error) {
       this.error(error as string | Error);
     }
   }
 
-  cleanFiles = (appName: string): void => {
-    fs.rmSync(`${appName}/.add-ons`, { recursive: true });
+  cleanFiles = async(appName: string): Promise<void> => {
+    this.log('Removing the .add-ons folder.');
+    return this.deleteAddOnsFOlder(appName);
+  };
+
+  deleteAddOnsFOlder = async(appName: string): Promise<void> => {
```

Comment:
I guess a typo maybe?
```suggestion
  deleteAddOnsFolder = async(appName: string): Promise<void> => {
```

Reply:
🙈  c0801c8